src/platforms/web/runtime/modules/attrs.js

```diff
@@ -83,12 +85,16 @@ function setAttr (el: Element, key: string, value: any) {
     } else {
       el.setAttributeNS(xlinkNS, key, value)
     }
+  } else if (isEmpty(key)) {
```

Comment:
this is a dev warning, it should not throw but warn **in dev only** like other warnings